gsod/README.md

```diff
@@ -42,7 +42,7 @@ There are fourteen fields recorded in the observation file. The `USAF` and `WBAN
 - **MaxGustSpeedMPH:** peak guest wind speed in miles per hour
 - **MaxTemperatureF:** maximum temperature, in Fahrenheit
 - **MinTemperatureF:** minimum temperature, in Fahrenheit
-- **PreciptitationInches:** total precipitation, in inches
+- **PrecipitationInches:** total precipitation, in inches
```

Comment:
This data is generated from scripts in `examples-private`.  Review those scripts for the spelling problem.

Reply:
Created a PR in that repo with Mike B as a reviewer to fix the spelling error